scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -108,7 +115,7 @@
         "workspaceLocation": "[replace(parameters('workspaceRegion'),' ', '')]",
         "dcrNameFull": "[Concat('MSCI', '-', variables('workspaceLocation'), '-', variables('clusterName'))]",
         "dcrName": "[if(greater(length(variables('dcrNameFull')), 64), substring(variables('dcrNameFull'), 0, 64), variables('dcrNameFull'))]",
-        "enableHighLogScaleMode": "[if(contains(parameters('streams'),'Microsoft-ContainerLogV2-HighScale'), bool('true'), bool('false'))]",
+        "enableHighLogScaleMode": "[if(or(contains(parameters('streams'),'Microsoft-ContainerLogV2-HighScale'), contains(parameters('streams'),'Microsoft-RetinaNetworkFlowLogs')), bool('true'), bool('false'))]",
```

Comment:
we should use "enableRetinaNetworkFlags" flag here instead of  'Microsoft-RetinaNetworkFlowLogs', right?

Reply:
changed